furusiya/model/maps/generators/forest_generator.py

```diff
@@ -86,7 +85,7 @@ def append_if_eligible(to_append):
         return explored
 
     def _fill_ground_holes(self):
-        start_position = self._find_empty_ground()
+        start_position = self._area_map.get_random_walkable_tile()
```

Comment:
There's a good chance this will end up generating a bunch of maps where 60-70% of the area is marked as "unreachable" and becomes impenetrable forest. The point of this was to pick a "reasonable" area, i.e. a 3x3 grid.

Did you test this by generating a bunch of forests to see if this is a problem or not?

Reply:
Didn't we already agree on discord that a 3x3 grid was problematic and that we should just make it get any empty tile?